### PR TITLE
change log REST endpoint with v3 prefixed

### PR DESCRIPTION
**Pull request by Shikugawa**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [ ] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

The previous name of log collection REST endpoint has inconsistency with other REST supported endpoint, such as `/v3/segment`. In this PR, we fix this problem.

### CODE REVIEW

**Comment by Shikugawa** [comment]
cc @kezhenxu94

**Review by wu-sheng — COMMENTED** [review]
Make sense to me. We missed that before. @kezhenxu94

**Comment by wu-sheng** [comment]
@kezhenxu94 Is this API new in 8.6? Whether we need changelog?

**Comment by kezhenxu94** [comment]
It' OK to me to add that prefix, the reason why I didn't add it is because for logging API, we don't have historical versions, (v1, v2).

This changes don't need change log as logging REST API is not released yet

File: docs/en/protocols/Log-Data-Protocol.md [hunk 1]
```diff
@@ -45,39 +45,37 @@ Json log record example:
...
-        {
-          "key": "logger",
-          "value": "com.example.MyLogger"
-        }
-      ]
-    },
-    "body": {
-      "text": {
-        "text": "log message"
+{
```
Comment (kezhenxu94) [inline docs/en/protocols/Log-Data-Protocol.md:53]: It's type should be array

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**